packages/contracts/src/core/dao/DAO.sol

```diff
@@ -110,6 +132,8 @@ contract DAO is
         address _trustedForwarder,
         string calldata daoURI_
     ) external initializer {
+        _reentrancyStatus = _NOT_ENTERED;
```

Comment:
As said before, having a zero-by default value might be simpler. If there are good reasons not to do it, feel free to resolve